I fixed logo.

sumosumo727 opened the pull request.

Commit:
I fixed logo.

Commit:
Merge branch 'main' into feature/Header_Footer

Commit:
Changed Logo and delete route "login" from web.php

kredo-school commented on resources/views/layouts/app.blade.php:
```diff
@@ -32,7 +32,8 @@
             <div class="container">
                 <a class="navbar-brand" href="{{ url('/') }}">
-                    <img class="logo" src="../storage/Logo_ExploreEase_temporaly.png" alt="Logo">
+                    <img class="logo" src="../storage/Logo.temporaly.png" alt="Logo" style="width:32px; height:32px;">
```
please include the image in your pull request because this implementation will only work on your end since you have the image in your local but won't work on our end. My suggestion for this is to create a directory inside `public` and name it `assets` and add the image file for the logo inside. and change the path in your `img` tag

kredo-school replied:
and also please resolve the conflicts indicated below.

sumosumo727 replied:
@kredo-school
Thank you for your comment. In regards to assets, do you mean this folder?
[assets folder.pdf](https://github.com/kredo-school/14th_explore_ease/files/13266386/assets.folder.pdf)

kredo-school replied:
@sumosumo727 that folder is for the build files for js and css
there should be a different folder relative to the `build` directory
should be `/public/assets/` then maybe also add another folder inside 
name the folder `images` and same the logo inside

sumosumo727 replied:
@kredo-school Thank you.
I put image file on "storage" folder, is it OK?
![image](https://github.com/kredo-school/14th_explore_ease/assets/130156131/55178d58-4a15-491b-8ba5-edae430533f3)

Also I change the path of the image file on "app.blade.php" like this.
![image](https://github.com/kredo-school/14th_explore_ease/assets/130156131/65926fd9-1806-4ed7-95af-92b46d7fc3e1)

then how can I commit and push these changes?
In "source control" on VS code, there is nothing relates to image or public/storage.
![image](https://github.com/kredo-school/14th_explore_ease/assets/130156131/987e07bf-f707-4c02-8f55-bd70d025f996)

Commit:
Changed place of storage of image.

Ditzler01 commented:
Please resolve the conflicts as well.

Commit:
Merge branch 'main' of https://github.com/kredo-school/14th_explore_ease into feature/Header_Footer

Commit:
Merge branch 'main' of https://github.com/kredo-school/14th_explore_ease into feature/Header_Footer

Commit:
[feature/Header_Footer] commit storage folder and
logo image
in public folder

Commit:
[feature/Header_Footer] comment out "storage/*key"
in .gitignore

Commit:
[feature/Header_Footer] edit .gitignore to push image file

Commit:
[feature/Header_Footer] delete .gitignore
in storage folder

Commit:
[feature/Header_Footer]
rename folder sotrage to asset

Commit:
[feature/Header_Footer]
delete storage folder
since it has little arrow somehow
create assets folder and move image to this folder

Ditzler01 committed:
Delete public/build directory
